client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -32,6 +35,15 @@ interface Props {
 
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth()
+
+    const demoUserSubmit = () => {
+      login("demo@hatchways.com", "demo123").then((data) => {
```

Comment:
I tried logging in as a Demo User with the above credentials, they don't seem to exist.
You can manually create a new user using Postman :)